src/main/java/model/Player.java

```diff
@@ -0,0 +1,45 @@
+package model;
+
+
+import exception.InvalidMoneyAmountException;
+
+public class Player extends Person{
+    private int wallet;
+    private int bet;
+    private int playerNumber;
+
+    public Player(){
+        wallet = 100;
+    }
+
+    /**
+     * @return the balance of the player's wallet
+     */
+    public int balance(){
```

Comment:
in Java typischerweise getBalance(). Die anderen getter und setter wurden auch so gemacht